package.json

```diff
@@ -3,6 +3,9 @@
   "scripts": {
     "pre-publish": "jlpm && cd buildutils && npm run build && cd .. && node buildutils/lib/prepublish.js",
     "add:sibling": "node buildutils/lib/add-sibling.js",
+    "analyze": "jlpm run analyze:dev",
+    "analyze:dev": "cd dev_mode && jlpm run build && open static/stats.html",
+    "analyze:prod": "cd dev_mode && jlpm run build:prod && open static/stats.html",
```

Comment:
Is there a platform independent version of `open`?

Reply:
It doesn't appear like there is: https://dwheeler.com/essays/open-files-urls.html

We could use an npm package like [opn](https://www.npmjs.com/package/opn)